front-end/src/App.tsx

```diff
@@ -1,6 +1,7 @@
 import { Provider } from "react-redux";
 import { ExampleFeature } from "./domains/example-feature/example-feature.index";
 import { store } from "./redux/store";
+import Footer from "./lib/Footer/Footer.component";
```

Comment:
If we export Footer.component correctly, you'll only need to write:

`import Footer from "./lib/Footer";` 

This is one of the many benefits of having an index file that exports the composition of all other elements in Footer.component.tsx 💡

Reply:
I'll make sure that is working correctly thank you for catching that!